api/models.go

```diff
@@ -200,15 +200,35 @@ type Transaction struct {
 	Type            string      `json:"type"`
 	Invoice         string      `json:"invoice"`
 	Description     string      `json:"description"`
-	DescriptionHash string      `json:"description_hash"`
+	DescriptionHash string      `json:"descriptionHash"`
 	Preimage        *string     `json:"preimage"`
-	PaymentHash     string      `json:"payment_hash"`
+	PaymentHash     string      `json:"paymentHash"`
 	Amount          uint64      `json:"amount"`
-	FeesPaid        uint64      `json:"fees_paid"`
-	CreatedAt       string      `json:"created_at"`
-	SettledAt       *string     `json:"settled_at"`
-	AppId           *uint       `json:"app_id"`
-	Metadata        interface{} `json:"metadata,omitempty"`
+	FeesPaid        uint64      `json:"feesPaid"`
+	CreatedAt       string      `json:"createdAt"`
+	SettledAt       *string     `json:"settledAt"`
+	AppId           *uint       `json:"appId"`
+	Metadata        Metadata    `json:"metadata,omitempty"`
+	Boostagram      *Boostagram `json:"boostagram,omitempty"`
+}
+
+type Metadata = map[string]interface{}
+
+type Boostagram struct {
+	AppName        string `json:"appName"`
+	Name           string `json:"name"`
+	Podcast        string `json:"podcast"`
+	URL            string `json:"url"`
+	Episode        string `json:"episode,omitempty"`
```

Comment:
@im-adithya why do some have omitempty others not? is it required?

Reply:
I took this from LBE repo [here](https://github.com/getAlby/lightning-browser-extension/blob/f1a7dd86fbad8e71d7095fc7507b55d79bdc86e6/src/types.ts#L957), whereever we have `?:` in the Boostagram type there, I've omitted

Reply:
but it looks completely arbitrary and doesn't seem to follow the spec - https://github.com/lightning/blips/blob/master/blip-0010.md#fields